src/Aspire.Cli/AppHostRunner.cs

```diff
@@ -40,7 +45,7 @@ public async Task<int> RunAppHostAsync(FileInfo appHostProjectFile, Cancellation
             process.Kill(false); // DCP should clean everything else up.
         }
 
-        await hostLifetime.StopAsync(cancellationToken).ConfigureAwait(false);
+        lifetime.StopApplication();
```

Comment:
Why is this required?

Reply:
Its not. Just tweaked things so process lifetime doesn't wait for RunAsync(...) to return (since we are going to tear down the process anyway).